src/app/components/Footer/__snapshots__/index.test.jsx.snap

```diff
@@ -2,25 +2,156 @@
 
 exports[`Footer should render correctly 1`] = `
 .c0 {
-  background-color: #404040;
+  background-color: #4C4C4C;
   font-family: ReithSansNewsRegular,Arial,Helvetica,freesans,sans-serif;
-  padding: 16px;
+  padding: 16px 8px;
 }
 
 .c1 {
+  border-bottom: 1px solid #FFFFFF;
+  display: grid;
+  grid-auto-flow: column;
+  grid-template-columns: repeat(2,50%);
+  grid-template-rows: repeat(4,50%);
+  list-style-type: none;
+  margin: 0 0 8px 0;
+  padding: 0 0 8px 0;
+}
+
+.c1 > li:first-child {
+  border-bottom: 1px solid #FFFFFF;
+  grid-column: 1/3;
+  margin-bottom: 8px;
+}
+
+.c2 {
+  min-width: 50%;
+  padding: 8px;
+}
+
+.c3 {
+  color: #FFFFFF;
+  font-family: ReithSansNewsBold,Arial,Helvetica,freesans,sans-serif;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+}
+
+.c3:hover,
+.c3focus {
+  -webkit-text-decoration: underline;
+  text-decoration: underline;
+}
+
+.c4 {
   color: #FFFFFF;
+  padding: 8px 0 0 0;
+  margin: 0;
+}
+
+@supports not (display:grid) {
+  .c1 > li:first-child {
+    width: 100%;
+  }
+}
+
+@supports not (display:grid) {
+  .c2 {
+    padding: 8px 0;
+    display: inline-block;
+  }
 }
 
 <footer
```

Comment:
@sareh should we be using a div here? As the footer will also need to contain the brand logo? Or will we add that to this component?

Reply:
@greenc05 Since this is the Footer component, the `<footer>` element should live here. Then when we can pull in the BBC News part in a later PR, via an `import` into this file. I think it makes sense to have the `footer` element live in the `Footer` component.

Reply:
Super